packages/next/build/swc/options.js

```diff
@@ -43,7 +43,11 @@ function getBaseSWCOptions({
         },
       },
     },
-    styledComponents: styledComponents ? {} : null,
+    styledComponents: styledComponents
+      ? {
+          displayName: Boolean(development),
```

Comment:
```suggestion
          displayName: development,
```
Should this always already be a boolean here?

Reply:
Given that `displayName: undefined` is interpreted as `true` (it's a default) so I wanted to make sure it does not unexpectedly enable itself in the future if the value changes